### ISSUE TITLE
Create static activities for select H5P activity types during print PDF export

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/ideas#263]
### Feature Description

Many of our users create H5P activities for use in their interactive webbooks. When we produce print PDF exports, we produce a fall-back block that lets users know there was an interactive activity in the book that couldn't be included in the print export but can be found at the chapter's URL. However, the initial state of some H5P activity types are still useful to readers of print texts. For this reason, some of our users have requested the ability to simple allow the initial state of certain H5P activities to be printed statically in PDF exports (in a similar way to the way that still shots from Youtube videos are included in their placeholders). Users have requested this feature for the following H5P activity types:
- [ ] Multiple Choice
- [ ] Fill in the Blank
- [ ] Mark the Words
- [ ] True / False
- [ ] Drag the Words
- [ ] Image Pairing

**Comment by cogdog** [comment]
Would also help if the URL shown in the PDF version linked to an anchor based on the iframe ID or maybe even a direct open of the H5P item- more details here http://kitchen.opened.ca/2021/01/19/h5p-in-print-pdf/

**Comment by SteelWagstaff** [comment]
@cogdog I agree that would be helpful. Have created a separate issue for that idea: https://github.com/pressbooks/ideas/issues/356. I suspect we might have go upstream to H5P's WP plugin to generate the necessary anchors, but will keep you updated as we dig in deeper and learn more.
